ledger/complete/mtrie/trie/trie.go

```diff
@@ -519,10 +520,9 @@ func SplitPaths(paths []ledger.Path, bitIndex int) int {
 	return i
 }
 
-// splitByPath permutes the input paths to be partitioned into 2 parts. The first part contains paths with a zero bit
-// at the input bitIndex, the second part contains paths with a one at the bitIndex. The index of partition
-// is returned.
-// The same permutation is applied to the proofs slice.
+// splitTrieProofsByPath permutes the input paths to be partitioned into 2 parts. The first part contains paths
+// with a zero bit at the input bitIndex, the second part contains paths with a one at the bitIndex. The index
+// of partition is returned. The same permutation is applied to the proofs slice.
```

Comment:
* fixed outdated method name in goDoc (`splitByPath` -> `splitTrieProofsByPath`)
* the rest is just breaking the lines differently